resourcecontrollerv2/resource_controller_v2_integration_test.go

```diff
@@ -272,21 +274,41 @@ var _ = Describe("Resource Controller - Integration Tests", func() {
 				shouldSkipTest()
 
 				options := service.NewListResourceInstancesOptions()
+				options.SetLimit(resultsPerPage)
 				headers := map[string]string{
 					"Transaction-ID": "rc-sdk-go-test03-" + transactionID,
 				}
 				options.SetHeaders(headers)
-				result, resp, err := service.ListResourceInstances(options)
 
-				//should return one or more instances
-				Expect(err).To(BeNil())
-				Expect(resp.StatusCode).To(Equal(200))
-				Expect(result).ToNot(BeNil())
+				results := []resourcecontrollerv2.ResourceInstance{}
 
-				fmt.Fprintf(GinkgoWriter, "ListResourceInstances() result:\n%s\n", common.ToJSON(result))
+				for {
+					result, resp, err := service.ListResourceInstances(options)
+
+					//should return one or more instances
+					Expect(err).To(BeNil())
+					Expect(resp.StatusCode).To(Equal(200))
+					Expect(result).ToNot(BeNil())
+
+					Expect(*result.RowsCount).Should(BeNumerically(">=", int64(1)))
+					Expect(*result.RowsCount).Should(BeNumerically("<=", int64(resultsPerPage)))
+					Expect(len(result.Resources)).Should(BeNumerically(">=", 1))
+					Expect(len(result.Resources)).Should(BeNumerically("<=", resultsPerPage))
+
+					results = append(results, result.Resources...)
+
+					if result.NextURL == nil {
+						break
+					}
+
```

Comment:
```suggestion
```
For situations where you use GetQueryParam(), you will get back a nil value if the input "next url" parameter is nil, so you can leverage that and simply the logic here a bit.   This applies to all places where you call GetQueryParam